server/Makefile

```
@@ -32,10 +32,11 @@ $(MOCKGEN): $(BIN_FOLDER)
 .PHONY: yq
 yq: $(YQ)
 $(YQ): $(BIN_FOLDER)
-	$(GO) build \
-		-modfile $(shell dirname $(ROOT_DIR))/hack/tools/yq/go.mod \
-		-o $(BIN_FOLDER)/yq \
-		github.com/mikefarah/yq/v4
+	test -s $(BIN_FOLDER)/yq || \
+		$(GO) build \
+			-modfile $(shell dirname $(ROOT_DIR))/hack/tools/yq/go.mod \
+			-o $(BIN_FOLDER)/yq \
+			github.com/mikefarah/yq/v4
```

After some playing around, I think this is safe to take out.

```suggestion
	$(GO) build \
		-modfile $(shell dirname $(ROOT_DIR))/hack/tools/yq/go.mod \
		-o $(BIN_FOLDER)/yq \
		github.com/mikefarah/yq/v4
```

Evidence: if you try to run `make yq` with `server/bin/yq` already existing, make refuses to do anything (it says there's nothing to be done).  If it doesn't exist, then it will always build, so the call to `test` isn't useful here.

I'll follow up with a PR later today to take out the other instances of this pattern we have (both here in `server/` and over in `operator/`.